pyqtgraph/examples/_RunExampleApp.py

```diff
@@ -4,7 +4,7 @@
 not run twice.
 """
 
-from ExampleApp import ExampleLoader
+from _ExampleApp import ExampleLoader
```

Comment:
I guess I should check - this doesn't break one of the various ways of running the example application does it?